crates/svm-runtime-c-api/build.rs

```diff
@@ -1,18 +1,22 @@
 extern crate cbindgen;
 use cbindgen::{Builder, Language};
-use std::{env, fs, path::PathBuf};
+use std::{env, path::PathBuf};
 
 fn main() {
     generate_svm_header();
 }
 
 fn generate_svm_header() {
     let crate_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
-
     let out_dir = env::var("OUT_DIR").unwrap();
+
+    // targeting the workspace 'target/release' dir
     let mut src_header = PathBuf::from(&out_dir);
-    src_header.push("svm");
-    src_header.set_extension("h");
+    src_header.pop();
+    src_header.pop();
+    src_header.pop();
+    src_header.pop();
+    src_header.push("release/svm.h");
```

Comment:
`release` means `production`. please try to use the `PROFILE` cargo env variable.
It seems that if you omit one `src_header.pop()` statement you can have `src_header.push("svm.h")`

Reply:
I thought to always put it under `release`, even if i’m coming from `debug` dir.
If to inspect `PROFILE`, what should be done in case of `debug`? put it under `debug` dir, or not do anything?

Reply:
I think the best will be to always generate `svm.h` into `PROFILE`.
Then each `example` can decide where to copy its assets from.